framework/components/AFeatureDiscovery/components/ATourTooltip.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import {AContainer, ARow, ACol} from "../../ALayout";
+import AButton from "../../AButton";
+import AIcon from "../../AIcon";
+
+const ATourTooltip = ({
+  index,
+  step,
+  isLastStep,
+  onPrev,
+  onClose,
+  onNext,
+  ...tooltipProps
+}) => {
+  return (
+    <div
+      {...tooltipProps}
+      className="a-tour-tooltip">
+      <AContainer fluid>
+        <ARow noGutters>
+          <ACol>
+            <div className="a-tour-header">
```

Comment:
Since these are child elements of the block, they need to be `a-tour-tooltip__header`, `a-tour-tooltip__body`, `a-tour-tooltip__footer`, `a-tour-tooltip__footer__right-buttons`